cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/SmartWorkflow.java

```diff
@@ -451,19 +452,22 @@ public void initialize(WorkflowContext context) throws Exception {
     postActions = new LinkedHashMap<>();
     spec = GSON.fromJson(context.getWorkflowSpecification().getProperty(Constants.PIPELINE_SPEC_KEY),
                          BatchPipelineSpec.class);
+    boolean isProcessTimingEnabled = SystemArguments.isProcessTimingEnabled(pipelineRuntime.getArguments().asMap(),
+                                                                            spec.isProcessTimingEnabled());
```

Comment:
nit: Change the variable name - having the same function name is a bit confusing